p2p/base/basic_packet_socket_factory.cc

```diff
@@ -62,7 +62,7 @@ AsyncPacketSocket* BasicPacketSocketFactory::CreateServerTcpSocket(
     int opts) {
   // Fail if TLS is required.
   if (opts & PacketSocketFactory::OPT_TLS) {
-    RTC_LOG(LS_ERROR) << "TLS support currently is not available.";
+    RTC_LOG(LS_INFO) << "TLS support currently is not available.";
```

Comment:
This one seems correct as it was, we should not be passing the TLS option if it's not available.